configure.sh

```diff
@@ -81,6 +81,9 @@ CMake Options (Advanced)
 Wasm Options
   --wasm=VALUE             set compilation extension for WebAssembly <WASM, JS or HTML>
   --wasm-flags='STR'       Emscripten flags used in the WebAssembly binary compilation
+  --wasm-web=CONFIG        use predefined web configuration for WASM compilation
+                           Available configurations:
```

Comment:
I would put a line here: `(takes precedence over ``--wasm`` and ``--wasm-flags)`